docs/feature-store/using-spark-engine.md

```diff
@@ -238,4 +268,73 @@ target = ParquetTarget(
 )
 
 fstore.ingest(feature_set, source, targets=[target], run_config=run_config, spark_context=spark_service_name)
-```
\ No newline at end of file
+```
+
+## Spark ingestion from Snowflake example
+
+Spark ingestion from Snowflake can be executed both remotely and locally. The following code executes local data ingestion from Snowflake.
+
+```
+from pyspark.sql import SparkSession
+
+import mlrun
+import mlrun.feature_store as fstore
+from mlrun.datastore.sources import SnowflakeSource
+
+spark = SparkSession.builder.appName("snowy").getOrCreate()
+
+mlrun.set_environment(project="feature-store")
+feature_set = fstore.FeatureSet(
+    name="customer", entities=[fstore.Entity("C_CUSTKEY")], engine="spark"
+)
+
+source = SnowflakeSource(
+    "customer_sf",
+    query="select * from customer limit 100000",
+    url="<url>",
+    user="nnnnnn",
+    password="nnnnnn",
+    database="SNOWFLAKE_SAMPLE_DATA",
+    schema="TPCH_SF1",
+    warehouse="compute_wh",
+)
+
+fstore.ingest(feature_set, source, spark_context=spark)
+```
+
+## Spark ingestion from Azure example
+
+Spark ingestion from Azure can be executed both remotely and locally. The following code executes remote data ingestion from Azure.
+
+```
+# Initialize the MLRun project object
+project_name = "spark-azure-test"
+project = mlrun.get_or_create_project(project_name, context="./")
+
+from mlrun.runtimes import RemoteSparkRuntime
+RemoteSparkRuntime.deploy_default_image()
+
+from mlrun.datastore.sources import CSVSource
+from mlrun.datastore.targets import ParquetTarget
+from mlrun import code_to_function
+import mlrun.feature_store as fstore
+
+feature_set = fs.FeatureSet("rides7", entities=[fs.Entity("ride_id")], engine="spark", timestamp_key="key")
+
+source = CSVSource("rides", path="wasbs://warroom@mlrunwarroom.blob.core.windows.net/ny_taxi_train_subset_ride_id.csv")
+
+spark_service_name = "spark-fs" # As configured & shown in the Iguazio dashboard
+
+fn = code_to_function(kind='remote-spark',  name='func')
+
+run_config = fstore.RunConfig(local=False, function=fn, handler="ingest_handler")
+
+target = ParquetTarget(partitioned = True, time_partitioning_granularity="month")
+
+feature_set.set_targets(targets=[target],with_defaults=False)
+
+fstore.ingest(feature_set, source, run_config=run_config, spark_context=spark_service_name)
+
+import mlrun
+import mlrun.feature_store as fs
```

Comment:
I think you misread my previous comment - I just meant for the code markers to be at the end of code. These lines belong at the beginning of the code. However, as noted in other comments - the 2nd line is redundant, just move the `import mlrun` line to the beginning.